vhost/Cargo.toml

```diff
@@ -23,7 +23,7 @@ vhost-net = ["vhost-kern"]
 vhost-user = []
 vhost-user-frontend = ["vhost-user"]
 vhost-user-backend = ["vhost-user"]
-gpu-socket = ["vhost-user"]
+gpu-socket = ["vhost-user", "vhost-user-backend"]
```

Comment:
Why do we need this dependency?
For example could we allow to use this crate only for frontend, but with gpu support ?

Reply:
Yes I think you are right, we don't really need this dependency.